commit 044ebd1d5a691b1638ae622994d02f74f6e43591
Author: Ken Wang <Qingqing.Wang@amd.com>
Date:   Thu Mar 17 17:26:57 2016 +0800

    drm/amdgpu: include the right version of gmc header files for iceland
    
    commit 16a8a49be1b878ef6dd5d1663d456e254e54ae3d upstream.
    
    Signed-off-by: Ken Wang <Qingqing.Wang@amd.com>
    Reviewed-by: Alex Deucher <alexander.deucher@amd.com>
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>

diff --git a/drivers/gpu/drm/amd/amdgpu/sdma_v2_4.c b/drivers/gpu/drm/amd/amdgpu/sdma_v2_4.c
index 2cf5018..b1c7a9b 100644
--- a/drivers/gpu/drm/amd/amdgpu/sdma_v2_4.c
+++ b/drivers/gpu/drm/amd/amdgpu/sdma_v2_4.c
@@ -32,8 +32,8 @@
 #include "oss/oss_2_4_d.h"
 #include "oss/oss_2_4_sh_mask.h"
 
-#include "gmc/gmc_8_1_d.h"
-#include "gmc/gmc_8_1_sh_mask.h"
+#include "gmc/gmc_7_1_d.h"
+#include "gmc/gmc_7_1_sh_mask.h"
 
 #include "gca/gfx_8_0_d.h"
 #include "gca/gfx_8_0_enum.h"
diff -uN a/1.txt b/1.txt
--- a/dummy/rpi_1399_044ebd1d5a691b1638ae622994d02f74f6e43591.txt	1970-01-01 00:00:00.000000000 +0000
+++ b/dummy/rpi_1399_044ebd1d5a691b1638ae622994d02f74f6e43591.txt	2013-12-23 04:07:40.000000000 +0000
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
